aws/src/database/dynamodb/scripts/contact_details_table/scan_and_patch_new_phone_numbers.py

```diff
@@ -0,0 +1,81 @@
+from dataclasses import dataclass
+
+from mypy_boto3_dynamodb.service_resource import Table
+
+#from aws.src.database.dynamodb.utils.get_by_secondary_index import get_by_secondary_index
+from aws.src.database.dynamodb.utils.get_dynamodb_table import get_dynamodb_table
+from aws.src.utils.csv_to_dict_list import csv_to_array
+from aws.src.utils.logger import Logger
+from aws.src.utils.progress_bar import ProgressBar
+from enums.enums import Stage
+from decimal import Decimal
+import time
+#
+# IMPORTANT:
+#
+# This script uses the scan() operation in DyanmoDB which is cery expensive
+# if used repeatedly. DO NOT invoke these methods in a loop. Rather, supply
+# a list of affected records and run the script once
+#
+
+@dataclass
+class Config:
+    TABLE_NAME = "ContactDetails"
+    LOGGER = Logger()
+    STAGE = Stage.HOUSING_DEVELOPMENT
+    FILE_PATH = "aws\src\database\input\\bad_phone_numbers.csv"
+    HEADING_FILTERS = {
+        "id": lambda x: bool(x),
+    }
+
+total_count = 0
+update_count = 0
+
+def process_item(item, bad_phone_numbers, dynamo_table, logger):
+    global update_count
```

Comment:
You shouldn't need this as it's already global from line 32
By convention you can write your globals in uppercase like `UPDATE_COUNT` to be clear that they're global in local scope